### PR TITLE
Add build failure notification

### PR DESCRIPTION
**Pull request by udda1996**
## Purpose
To send a message to google chat to notify the build failures in the connector

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: .github/workflows/daily-build.yml [hunk 1]
```diff
@@ -71,15 +71,11 @@ jobs:
               env:
                 JAVA_HOME: /usr/lib/jvm/default-jvm
                 JAVA_OPTS: -DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true
-            
-            # Publish Github Package
-            - name: Publish Github Package
```
Comment (keizer619) [inline .github/workflows/daily-build.yml:76]: We need this for release publishing
@udda1996

Comment (keizer619) [reply .github/workflows/daily-build.yml:76]: this is fine as we use central stage instead of GH packages

Comment (LakshanSS) [reply .github/workflows/daily-build.yml:76]: Yes. We need to remove this publishing as github packages part.

**Review by LakshanSS — APPROVED** [review]

**Comment by LakshanSS** [comment]
Looks good for me

File: .github/workflows/daily-build.yml [hunk 2]
```diff
@@ -71,15 +71,17 @@ jobs:
...
\ No newline at end of file
+                curl \
+                -X POST 'https://chat.googleapis.com/v1/spaces/${{secrets.BALLERINA_CHAT_ID}}/messages?key=${{secrets.BALLERINA_CHAT_KEY}}&token=${{secrets.BALLERINA_CHAT_TOKEN}}' \
+                --header 'Content-Type: application/json' \
+                -d '{"text": "*module-ballerinax-mongodb daily build failure* \nPlease \n visit <https://github.com/ballerina-platform/module-ballerinax-mongodb/actions?query=workflow%3A%22Daily+build%22|the daily build page> for more information"}'
+
+                curl \
+                -X POST 'https://chat.googleapis.com/v1/spaces/${{secrets.CONNECTOR_CHAT_ID}}/messages?key=${{secrets.CONNECTOR_CHAT_KEY}}&token=${{secrets.CONNECTOR_CHAT_TOKEN}}' \
+                --header 'Content-Type: application/json' \
+                -d '{"text": "*module-ballerinax-mongodb daily build failure* \nPlease \n visit <https://github.com/ballerina-platform/module-ballerinax-mongodb/actions?query=workflow%3A%22Daily+build%22|the daily build page> for more information"}'
```
Comment (LakshanSS) [inline .github/workflows/daily-build.yml:87]: Shall we make only the `module-ballerinax-mongodb` as bold and keep the `daily build failure` as normal text?

And remove the `\n` between the Please and visit

**Review by keizer619 — APPROVED** [review]

**Pull request merged**